install.sh

```diff
@@ -184,7 +184,7 @@ if [[ $no_modify_config ]] && [[ $append_to_config ]]; then
 	exit 1
 fi
 
-BASH_IT="$(cd "$(dirname "$0")" && pwd)"
+BASH_IT="${BASH_SOURCE%/*}"
```

Comment:
this gives us the relative path of bash-it and not the absolute one. it works for the installation, but lets at least add a comment about it here (or change to absolute path like before)

Reply:
I think add a comment to the template `.bash_profile`, sound good?

Reply:
Except we modify the template during installation by replacing `{{BASH_IT}}`, so it makes more sense to make sure this one is absolute. ok